modules/monitoring-supported-remote-write-authentication-settings.adoc

```diff
@@ -0,0 +1,314 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: REFERENCE
+[id="supported_remote_write_authentication_settings_{context}"]
+= Supported remote write authentication settings
+
+You can use different methods to authenticate with a remote write endpoint.
+Currently supported authentication methods are AWS Signature Version 4, basic authentication, bearer token authorization, OAuth 2.0, and TLS client.
+The following table provides details about supported authentication methods for use with remote write.
+
+[options="header"]
+|===
+
+|Authentication method|Config map field|Description
+
+|AWS Signature Version 4|`sigv4`|This method uses AWS Signature Version 4 authentication to sign requests.
+You cannot use this method simultaneously with bearer token, OAuth 2.0, or basic authentication.
+
+|basic authentication|`basicAuth`|Basic authentication sets the authorization header on every remote write request with the configured username and password.
+
+|bearer token authorization|`Bearer`|Bearer token authentication sets the authorization header on every remote write request using the configured token.
+You can configure bearer token authentication by specifying a secret that contains the bearer token information. 
+When you use this method, the `remoteWrite` settings in the config map must include an `authorization` `type` of `Bearer`.
+
+|OAuth 2.0|`oauth2`|An OAuth 2.0 configuration uses the client credentials grant type. 
+Prometheus fetches an access token from `tokenUrl` with the specified client ID and client secret to access the remote write endpoint using this token.
```

Comment:
```suggestion
Prometheus fetches an access token from `tokenUrl` with the specified client ID and client secret to access the remote write endpoint.
```

Reply:
deleted the final phrase per your suggestion